java/webrtc-hello-world/src/main/java/com/bandwidth/webrtc/examples/helloworld/HelloWorldController.java

```diff
@@ -130,14 +146,14 @@ public HelloWorldController(AccountProperties accountProperties, VoiceProperties
         logger.info("received answered callback for call {} to {}", callId, body.get("to"));
 
         // Retrieve the participant and token that we set for this call id in initiateCall below
-        CreateParticipantResponse response = calls.get(callId);
+        AccountsParticipantsResponse response = calls.get(callId);
         if (response == null) {
             logger.warn("no participant found for call {}", callId);
             return ResponseEntity.badRequest().build();
         }
 
         // Generate transfer BXML from the device token and return that to the Voice API
-        String transferBxml = webRtcClient.generateTransferBxml(response.getToken());
+        String transferBxml = generateTransferBxml(response.getToken());
```

Comment:
so just to be clear, we transfer the call on both an incoming call and the call being answered?  Is this a well-known pattern?